src/components/Dashboard/Form/FieldWrapper/SelectOptions.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const Opts = props => {
+  return (
+    <select name={props.name} onChange={props.onChange}>
+      {props.options.map(x => {
```

Comment:
Before return I recommend props destructuring
```suggestion
const { name, onChange, options } = props
```